apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -45,7 +45,7 @@ const TeacherHomepage = React.createClass({
         <RecentCourses
           courses={courses}
           showAllCoursesLink={true}
-          heading={i18n.recentCourses()}
```

Comment:
This probably means recentCourses isn't used anywhere and we can delete that string?